### PR TITLE
Zoom.us for IT

### PR DESCRIPTION
**Pull request by targendaz2**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Review by ran-dall — APPROVED** [review]

File: Casks/zoomus-it.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
+cask 'zoomus-it' do
+  version '5.0.26223.0603'
+  sha256 '15612fac01df6e3ac560e217758826d9cdb3d2caa843b173bb3340a54fe76a2c'
+
+  # d11yldzmag5yn.cloudfront.net/ was verified as official when first introduced to the cask
+  url "https://d11yldzmag5yn.cloudfront.net/prod/#{version}/ZoomInstallerIT.pkg"
+  appcast 'https://zoom.us/download'
+  name 'Zoom.us for IT'
+  homepage 'https://www.zoom.us/'
```
Comment (vitorgalvao) [inline Casks/zoomus-it.rb:9]: This is the general webpage. Isn’t there a page for this specific version?

Comment (targendaz2) [reply Casks/zoomus-it.rb:9]: Hey, there's no specific homepage for this version of Zoom.us, similar to the "zoomus-outlook-plugin" cask.

Comment (ran-dall) [reply Casks/zoomus-it.rb:9]: `https://support.zoom.us/hc/en-us/articles/115001799006-Mass-Deployment-with-Preconfigured-Settings-for-Mac` is the closest thing I could find to a homepage.

@targendaz2 Also, I believe the should `name` is `Zoom for IT Admins`? I feel `Zoom for IT` is ambiguous when the technical name is `Zoom Client for IT Admins for Mac`.

Comment (targendaz2) [reply Casks/zoomus-it.rb:9]: I can update the cask with that. Do you have a recommendation for the cask name? `zoom-for-it-admins`?

Comment (ran-dall) [reply Casks/zoomus-it.rb:9]: @targendaz2 `zoom-for-it-admins` LGTM 👍.

**Comment by targendaz2** [comment]
Just implemented those changes to the cask name and homepage. Let me know if there's anything else I should do!

**Comment by core-code** [comment]
if this app is always updated in lock-step with the main zoomus cask it might make sense to adopt the appcast we use there:

`  appcast 'https://macupdater.net/cgi-bin/check_urls/check_url_redirect.cgi?url=https://zoom.us/client/latest/Zoom.pkg'
`

**Comment by vitorgalvao** [comment]
> if this app is always updated in lock-step with the main zoomus cask it might make sense to adopt the appcast we use there:

Good point. Is it, @targendaz2?

**Comment by targendaz2** [comment]
It is updated in lock-step with zoomus. I've updated the appcast as suggested in pull request #85284

**Pull request merged**